src/exprestive.coffee

```diff
@@ -51,7 +51,8 @@ class Exprestive
 
   # Middleware to set reverse routes on req.locals
   setReverseRoutesOnReqLocals: (req, res, next) =>
-    res.locals.routes = @reverseRoutes
+    if res.locals?
+      res.locals.routes = @reverseRoutes
```

Comment:
Put the if-clause after the expression? It's not really code flow, just to guard against an edge case, so doesn't have to be so front and center.

```coffeescript
res.locals.routes = @reverseRoutes if res.locals?